src/main/java/frc/team2412/robot/commands/auto/AutoCommandPickerCommand.java

```diff
@@ -43,38 +45,38 @@ public AutoCommandPickerCommand(DriveBaseSubsystem driveBaseSubsystem, IntakeOnO
 
 	@Override
 	public void execute() {
-		
-		Command command;
+
 		if (RobotState.sixBallAuto == true) {
-			
-			command = new SixBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem,
-					m_intakeUpDownSubsystem, m_liftSubsystem, m_flywheelSubsystem, m_indexerSensorSubsystem,
-					m_indexerMotorSubsystem, m_hoodSubsystem);
-			CommandScheduler.getInstance()
-					.schedule(new SixBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem,
-							m_intakeUpDownSubsystem, m_liftSubsystem, m_flywheelSubsystem, m_indexerSensorSubsystem,
-							m_indexerMotorSubsystem, m_hoodSubsystem));
+
+			command = new SixBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem, m_intakeUpDownSubsystem,
+					m_liftSubsystem, m_flywheelSubsystem, m_indexerSensorSubsystem, m_indexerMotorSubsystem,
+					m_hoodSubsystem);
+
 		} else if (RobotState.threeBallAuto == true) {
-			
-			command = new SixBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem,
-					m_intakeUpDownSubsystem, m_liftSubsystem, m_flywheelSubsystem, m_indexerSensorSubsystem,
-					m_indexerMotorSubsystem, m_hoodSubsystem);
-			
-			CommandScheduler.getInstance()
-			.schedule(new ThreeBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem,
-					m_intakeUpDownSubsystem, m_liftSubsystem, m_flywheelSubsystem, m_indexerSensorSubsystem,
-					m_indexerMotorSubsystem, m_hoodSubsystem));
+
+			command = new SixBallAutoCommandGroup(m_driveBaseSubsystem, m_intakeOnOffSubsystem, m_intakeUpDownSubsystem,

```

Comment:
? This used to schedule a ThreeBallAutoCommandGroup, and this is in the `if (RobotState.threeBallAuto == true)` clause